hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -226,10 +226,22 @@ public function testFTECalculationOnCasualTypeAndEmptyAmount() {
     $importEmptyContractResponse = $this->runImport($emptyHoursContract);
     $this->assertEquals(CRM_Import_Parser::VALID, $importEmptyContractResponse);
 
+    // Only Mandatory Fields
```

Comment:
With this new addition, the method name doesn't reflect what it's being tested anymore. The options here are either:
- Split the test into 3 tests (one for casual type, one for empty amount and one for mandatory fields)
- Leave the existing test alone and create a separate one just for the mandatory fields
- Update the test name to reflect what is being tested now

My order of preference if from the first option to the last one. Honestly, the last option should only be used in case the performance impact of adding a separate test is too high. Otherwise, it's always best to make each test to assert a single scenario.